.husky/post-checkout

```diff
@@ -1,4 +1,2 @@
 #!/usr/bin/env sh
-. "$(dirname -- "$0")/_/husky.sh"
```

Comment:
> also remove refs to _/husky.sh which seem to have been deleted upstream

This is till there, `husky` is one of the `devDependencies`. This shell script is installed with it.

Reply:
weird, it's no longer on my filesystem. I had to remove these in order to make the PR preflight checks pass.